apps/next/package.json

```diff
@@ -11,6 +11,7 @@
     "build:temporal": "tsc --build ./temporal/tsconfig.json",
     "build": "concurrently --raw \"npm:build:next\" \"npm:build:temporal\"",
     "db:push": "source $HOME/.nvm/nvm.sh; nvm use 20; drizzle-kit push:pg",
+    "ci:db:push": "drizzle-kit push:pg",
```

Comment:
looking at this i think it was a mistake to add the `source $HOME/.nvm/nvm.sh; nvm use 20;` to the db:push script. I added it because drizzle-kit requires node (can't run w/ bun), but we can just assume you have `node` available. Can you remove the nvm stuff from db:push and just use regular db:push without introducing the ci:db:push?

Reply:
Done.